src/main/java/io/pakland/mdas/githubstats/domain/Organization.java

```diff
@@ -0,0 +1,46 @@
+package io.pakland.mdas.githubstats.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+/**
+ * <b>Class</b>: organization <br/>
+ * <b>Copyright</b>: 2022 La Salle - mdas <br/>.
+ *
+ * @author 2022  MDAS <br/>
+ * <u>Service Provider</u>: La Salle <br/>
+ * <u>Developed by</u>: mdas <br/>
+ * <u>Changes:</u><br/>
+ * <ul>
+ *   <li>
+ *     Septiembre 13, 2022 Creación de Clase.
+ *   </li>
+ * </ul>
+ */
```

Comment:
Same for this and all next... 😅

Reply:
Is one of the checkstyle metrics on backend --> https://checkstyle.sourceforge.io/config_javadoc.html

Reply:
Right I see. From one side we could disable this rule, which I believe it should be possible. On the other side, I think that what is written _in this case_ do not add any value, buf javadoc will for sure add value when describing arguments of a function or the class spec. Therefore, we could add docs about that class, yet I think that all this metadata provided is not necessary as we do not intend to publish as a public library. Just to simplify our work.

Cc: @plozanol @sdomingobasora @manerow

Reply:
Hi, looking at the requirements of the project i dont find documenting the domain is necessary for now, for me it seems self-explainatory enough. As @mikededo stated, the fields documented here don't add seem to add any value. Also, using the UML schema and Flow diagram from the project documentation along with the list of issues and their respective definitions should do for presenting our solution in class. 
Nevertheless, i do consider we could use Javadoc for more specific scenarios like documenting our usecase or repository methods. ([Documenting with Javadoc at method level](baeldung.com/javadoc#4-javadoc-at-method-level))

Reply:
Agreed with @mikededo and @manerow and nothing to add.

As suggested we should keep Javadoc as the documentation method for the other classes

Reply:
Then @cjp4 disable this rule so that we do not need to add extra documentation 💯